rebench.conf

```diff
@@ -44,8 +44,12 @@ benchmark_suites:
 executors:
     Lua:
         executable: lua
+        path: yklua/src/
```

Comment:
Should this be `lua/src` or ...?

Reply:
`yklua/src` is correct.

Reply:
Oh, hang on: this means we're not benchmarking *lua* but *yklua* in two different configs. Note that "yklua without yk" will not, over time, be quite the same thing as "yk". Right now, if you don't `#define USE_YK` you should end up (more-or-less) with the same binary, but I think that's not going to end up being viable when we start doing things like rewriting functions to expose elidable functions etc.

That suggests to me that we should build a "normal" Lua, from source, that's not derived directly from yklua.

Reply:
I thought it would be unfair to not use the same source repo, but now I'm not so sure.

I was working under the assumption that all code we added to the lua source code is guarded with `USE_YK`.

If that's not true we should indeed download a separate "no yk changes in sight" source tree and use that as a baseline.

Maybe we should just do that anyway for peace of mind. OK to do that in another PR?

Reply:
> I was working under the assumption that all code we added to the lua source code is guarded with USE_YK.

It is *right now* (probably...) but I doubt that will be the case going forwards. [I'm actually unsure whether `USE_YK` is massively useful even now... it does help a bit with reading the code, but you can always diff against "true Lua" to see the diffs, which is probably a better thing to do anway.]

>If that's not true we should indeed download a separate "no yk changes in sight" source tree and use that as a baseline.
>
>Maybe we should just do that anyway for peace of mind. OK to do that in another PR?

Yes and yes respectively.